media/script/main.js

```diff
@@ -0,0 +1,54 @@
+const menuBars = document.querySelector('.menu-bars');
+const main = document.querySelector('.main-body');
+
+const menuDiv = document.createElement('div');
+menuDiv.className = 'toggle-nav';
+const menuUl = document.createElement('ul');
+const menuPortfolio = document.createElement('li');
+menuPortfolio.textContent = 'Portfolio';
+const menuAbout = document.createElement('li');
+menuAbout.textContent = 'About';
+const menuContact = document.createElement('li');
+menuContact.textContent = 'Contact';
+const closeBtn = document.createElement('span');
+closeBtn.textContent = 'x';
+closeBtn.id = 'toggle-close';
+closeBtn.style.color = 'white';
+
+menuUl.appendChild(menuPortfolio);
+menuUl.appendChild(menuAbout);
+menuUl.appendChild(menuContact);
+
+menuDiv.appendChild(closeBtn);
+menuDiv.appendChild(menuUl);
+
+menuBars.addEventListener('click', () => {
+  main.prepend(menuDiv);
+  menuBars.style.display = 'none';
+  menuDiv.style.display = 'block';
+  menuDiv.style.backgroundColor = '#3C3A39';
+  menuDiv.style.height = '100vh';
+});
```

Comment:
[OPTIONAL] ♻
The mobile menu click event listener works perfectly, but if you resize the window with the menu still open, the menu won't go away and will cover the whole page. Now, this won't happen on most devices, but it could happen for example if a user is in vertical mode on a mobile phone and switches to landscape mode. You could add a function to check the window size and display the menu only if a certain size is met, or add a media query to the css.